### PR TITLE
refresh token lifetime clarifications

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** added content around rotating refresh tokens and what that means to refresh token lifetime and how it differs from the idle time
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-416277](https://oktainc.atlassian.net/browse/OKTA-416277)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md](https://github.com/okta/okta-developer-docs/blob/48ef6634371cbb8aeb2db99d5e845f64a92ddb7b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-authz-server%2Fmain%2Findex.md) | 79 | 83 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9a533a72-242d-4c24-a485-0d193a27a6b7) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.b0f38940-e38a-46de-8c89-f58ce4a4147e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md](https://github.com/okta/okta-developer-docs/blob/b7171c079c69626e87482ed7ae5dadfbaf90c476/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-authz-server%2Fmain%2Findex.md) | 79 | 84 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5faa584c-0a74-4ebe-b179-1cfce7873fe9) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.992898d3-c7a2-438b-bf10-0ae26fddcce8?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by shantanusardal-okta — APPROVED** [review]

**Comment by susanharper-okta** [comment]
Also approved by Venkat Viswanathan on Slack

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md [hunk 1]
```diff
@@ -86,8 +86,14 @@ To create a rule for a policy:
...
-    * **AND Refresh token lifetime is:** Select the length of time before a refresh token expires.
-    * **but will expire if not used every:** Defines when the refresh token expires if it isn't used. Leave the default of **7 days** or make any necessary changes.
+    * **AND Refresh token lifetime is:** Leave the default of `Unlimited` unless you need to customize how long you can use a refresh token before it expires.
+
+    > **Note:** If you customize the refresh token lifetime, and you [rotate your refresh tokens](/docs/guides/refresh-tokens/main/#refresh-token-rotation), that lifetime is inherited from the initial refresh token minted when the user first authenticates and stays the same through the series of refresh token rotations until the timeframe that you set expires.
+
+    * **but will expire if not used every:** Defines the maximum time that a refresh token can be idle (unused) before it expires. For example, if a request isn't made to refresh an access token within this timeframe, the refresh token expires. Leave the default of **7 days** or make any necessary changes.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:93]:
```suggestion
    * **but will expire if not used every:** Defines the maximum time that a refresh token can be idle (unused) before it expires. Leave the default of **7 days** or make any necessary changes.
```

Comment (grahamsmith-okta) [reply packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:93]: Second sentence seems unnecessary?

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md [hunk 2]
```diff
@@ -86,8 +86,14 @@ To create a rule for a policy:
...
     * **THEN Use this inline hook:**  Select an [inline hook](/docs/concepts/inline-hooks/), if any, that you want to use to customize the token returned by Okta API Access Management.
     * **AND Access token lifetime is:** Select the length of time before an access token expires.
-    * **AND Refresh token lifetime is:** Select the length of time before a refresh token expires.
-    * **but will expire if not used every:** Defines when the refresh token expires if it isn't used. Leave the default of **7 days** or make any necessary changes.
+    * **AND Refresh token lifetime is:** Leave the default of `Unlimited` unless you need to customize how long you can use a refresh token before it expires.
+
+    > **Note:** If you customize the refresh token lifetime, and you [rotate your refresh tokens](/docs/guides/refresh-tokens/main/#refresh-token-rotation), that lifetime is inherited from the initial refresh token minted when the user first authenticates and stays the same through the series of refresh token rotations until the timeframe that you set expires.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:91]:
```suggestion
    > **Note:** If you customize the refresh token lifetime and [rotate your refresh tokens](/docs/guides/refresh-tokens/main/#refresh-token-rotation), that lifetime is inherited from the initial refresh token minted when the user first authenticates. It stays the same through the series of refresh token rotations until the timeframe that you set expires.
```

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md [hunk 3]
```diff
@@ -86,8 +86,14 @@ To create a rule for a policy:
...
+
+    * **but will expire if not used every:** Defines the maximum time that a refresh token can be idle (unused) before it expires. For example, if a request isn't made to refresh an access token within this timeframe, the refresh token expires. Leave the default of **7 days** or make any necessary changes.
+
+    > **Note:** This idle time resets each time a request is made to refresh an access token and when a refresh token is rotated. It doesn't affect the refresh token lifetime, only how long the refresh token can remain unused before it expires.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:95]:
```suggestion
    > **Note:** This idle time resets each time a request is made to refresh an access token and when a refresh token is rotated. It doesn't affect the refresh token lifetime.
```

Comment (grahamsmith-okta) [reply packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:95]: "only" phrase is a little awkward, may be clearer to just cut it (already stated in the previous definition anyway)

**Review by grahamsmith-okta — COMMENTED** [review]
A couple of suggestions for shortening/clarity. Otherwise good to go!

**Review by grahamsmith-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md](https://github.com/okta/okta-developer-docs/blob/b71b9f9c2e78af43bf0fcd1874a4b692d515ecaf/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-authz-server%2Fmain%2Findex.md) | 79 | 86 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fa4b2c4a-787d-4bf4-9c0f-a0f01c5d1c69) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.514d7b12-33d1-4ded-b683-9e66a1a1aee2?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md](https://github.com/okta/okta-developer-docs/blob/8db89618aefb3ec1fb3214f3f355eb7cc815102d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-authz-server%2Fmain%2Findex.md) | 88 | 42 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8b0c863c-c65f-428d-89af-768ed3056aba) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.b1fb3103-7365-4ec9-8de0-d0751fe6cb0e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Pull request merged**